src/test/java/com/ixortalk/organization/api/rest/OrganizationAdminController_RemoveAdminRights_IntegrationAndRestDocTest.java

```diff
@@ -165,27 +165,5 @@ public void userDoesNotExist() {
         verify(auth0Roles, never()).removeRolesFromUser(anyString(), anySetOf(String.class));
```

Comment:
Since there is no Auth0 interaction at all anymore, maybe best to remove them `verify`'s as well?